tests/integration/sdk_api/alerts/test_alerts.py

```diff
@@ -26,6 +26,14 @@
 
 
 class TestAlerts(tests.integration.sdk_api.base.TestMLRunIntegration):
+    @classmethod
+    def setup_class(cls):
+        cls.extra_env = cls.extra_env or {}
+        cls.extra_env["MLRUN_ALERTS__MODE"] = (
+            mlrun.common.schemas.alert.AlertsModes.enabled
+        )
+        super().setup_class()
```

Comment:
This call is likely overwriting or ignoring cls.extra_env if it's being set after the parent class setup logic..
It should be called at the beginning of this flow and not at the end of it

Reply:
Yes I changed it to only set the `extra_env` class property and not override `setup_class`.